.github/workflows/maven-ci.yml

```diff
@@ -13,16 +13,24 @@ jobs:
         with:
           java-version: '16'
           distribution: 'adopt'
+      
+      # Github Authentication is required to download artifacts from github packages
+      # A secret is created for this repo, so that github actions can fetch it, the secret is named after GH_PAT_FOR_ACTIONS_TOKEN
+      - name: maven-settings
+        uses: s4u/maven-settings-action@v2
+        with:
+          servers: '[{"id": "github", "username": "madieUser", "password": "${GITHUB_TOKEN_REF}"}]'
+          githubServer: false
+
       - name: Build with Maven
-        run: mvn --batch-mode --update-snapshots verify
-      #- uses: actions/checkout@master
-      - name: List files in the repository
-        run: |
-          ls ${{ github.workspace }}
-      - uses: codecov/codecov-action@v2
+        run: mvn clean install --file pom.xml
```

Comment:
run: mvn --batch-mode --update-snapshots verify
I think we still want the mvn command like this.  We want to make sure that snapshots are updated, and there isn't any reason to do "install" on this package.